src/backend/api/v1/index.ts

```diff
@@ -2,8 +2,10 @@ import express from 'express';
 import {chatsRouter} from "./Chats";
 import {usersRouter} from "./Users";
 import {postsRouter} from "./Posts";
+import {commentsRouter} from "./Comments";
 export const apiV1Router = express.Router();
 
 apiV1Router.use('/chats', chatsRouter);
 apiV1Router.use('/users', usersRouter);
 apiV1Router.use('/posts', postsRouter);
+apiV1Router.use('/comments', commentsRouter);
```

Comment:
You don't need to add this here. I think I added it inside PostRouter. So if you submit a new comment it would be to `/posts/comments` because comment needs to be inside a post.

Reply:
But I couldn't use comment-related APIs without it

Reply:
You tried `/api/v1/posts/comments`? Im pretty sure it works if u look inside the CommentTests

Reply:
oh, didnt try that